src/hooks/useAutorisation.js

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+import useToken from './useToken';
+
+function useAutorisation() {
+  useEffect(() => {
+    if (!localStorage.getItem('TOKEN')) {
```

Comment:
the 'TOKEN' string is a good example of something that should be declared as a constant at app level and used everywhere you have that string. It's very easy to do a typo and spend a big amount of time trying to understand what broke.